tests/TestFormula.elm

```diff
@@ -202,6 +203,91 @@ parseSignedTests =
         , test "F a" <| \() -> testParseSigned "F a" <| F a
         ]
 
-
+--testovanie spravneho dosadenia
+testSubstitution : Substitution -> Formula -> Formula -> Test
+testSubstitution s original new =
+    test
+        ("in formula " ++ (original |> Formula.strFormula) ++" using subs " ++ (Formula.strSubstitution s))
+        (\() -> Expect.equal (Ok new) (Formula.removeQuantifierAndSubstitute s original))
```

Comment:
Tieto testy by mali testovat  `substFormula` (reps `substitute` ked to premenujes).
Samozrejme mozes spravit aj nejake testy na to specialne spravanie `removeQuanti.....`